main/roles/setup_configuration/config.toml

```diff
@@ -291,7 +291,7 @@ enable = false
 #
 # For Cosmos SDK-based chains, trust_period should usually be about 2/3 of the unbonding time (~2
 # weeks) during which they can be financially punished (slashed) for misbehavior.
-rpc_servers = "157.230.110.18:26657,46.101.146.48:26657"
+rpc_servers = "{{ trust_rpc1 }},{{ trust_rpc2 }}"
```

Comment:
You can create a unique variable before use it: if one day we add a rpc server it could be useful